spec/ttml1.xml

```diff
@@ -7682,6 +7682,10 @@ to satisfy alignment in the block progression dimension. For example, this assur
 the background color associated with the <el>body</el> element, if not <code>transparent</code>,
  will fill the containing region wholly.</p>
 </note>
+<note role="clarification">
+<p>For the purpose of performing presentation processing, the <code>auto</code> value of
+the XSL FO <code>overflow</code> property is interpreted as <code>visible</code>.</p>
+</note>
 </div3>
```

Comment:
Text should be added why this clarification is needed e.g.

"The overflow property of a fo:block element that have been mapped from body, div or p has the initial value of auto."

Reply:
Really? In general, we don't provide motivation or rationale information, which is more appropriate in a user guide than a technical specification. Furthermore, here, we (have to) assume that the reader is familiar with XSL-FO. Further still, and more importantly, it is not true that "the overflow property of a fo:block element" [has been] "mapped from body, div, or p", since tts:overflow does not apply to the latter.

Reply:
> In general, we don't provide motivation or rationale information, which is more appropriate in a user guide than a technical specification.

As our discussion has shown clarification is more than needed.

> Furthermore, here, we (have to) assume that the reader is familiar with XSL-FO. 

Unfortunately we can not assume that the reader will be familiar with XSL-FO to that level of detail.

> Further still, and more importantly, it is not true that "the overflow property of a fo:block element" [has been] "mapped from body, div, or p", since tts:overflow does not apply to the latter.

Ok, than we can re-word it. It is not the overflow property that is mapped but body, div and p are mappped to an fo:block element. This fo:block element has an initial value for the overflow property. 

Proposal:

 "The fo:block element that have been mapped from body, div or p has an overflow property with the initial value of auto."

Reply:
Upon review of XSL-FO 1.1, Section B.4, we see that overflow does not apply to fo:block or fo:inline.

> Property overflow applies to: fo:region-body, fo:region-before, fo:region-after, fo:region-start, fo:region-end, fo:block-container, fo:external-graphic, fo:instream-foreign-object, and fo:inline-container.

-